### DISCUSSION TITLE
Connecting to server on Microsoft SQL Server Management Studio

### DISCUSSION THREAD

**Discussion by gaurav031295** [FreeOpcUa/opcua-asyncio#349] [General]
I have a server running on the Microsoft SQL Server Management Studio. Does anyone know if I can connect to this server as a client using the asyncua package ? The client code specifies the url with the IP address and port number. I tried to enter that by checking the Server Configuration Manager. However, I get a timeout error. So the client never connects.

I think I am getting the concept wrong because I am not sure if it is possible to connect to a server in Server Management Studio as a client. I can successfully connect to the server and write queries with the pyodbc package.

Please Advise!

**Comment by oroulet** [comment]
has that server an opcua interface? azure has something,,,

**Comment by gaurav031295** [comment]
That is exactly something I am not aware of! I am also not sure how to check it!

**Comment by swamper123** [comment]
Well either you have to give a call to the people who provide that SQL Server (Support Hotline or similar) or make your own researches to that topic.

Since our Client works with a bunch of the OPC UA Servers, it may be a reason that this program does not support OPC UA.
